extensions/participant-store-spi/build.gradle.kts

```diff
@@ -1,15 +1,25 @@
 plugins {
     `java-library`
+    `java-test-fixtures`
 }
 
 val jacksonVersion: String by project
 val jetBrainsAnnotationsVersion: String by project
+val jupiterVersion: String by project
+val assertj: String by project
 
 dependencies {
     api(project(":extensions:dataspace-authority-spi"))
     api("org.jetbrains:annotations:${jetBrainsAnnotationsVersion}")
 
     implementation("com.fasterxml.jackson.core:jackson-databind:$jacksonVersion")
+
+    testFixturesImplementation(testFixtures(project(":extensions:dataspace-authority-spi")))
+    testFixturesImplementation("org.junit.jupiter:junit-jupiter-api:${jupiterVersion}")
+    testFixturesImplementation("org.junit.jupiter:junit-jupiter-params:${jupiterVersion}")
+    testFixturesRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:${jupiterVersion}")
+    testFixturesImplementation("org.assertj:assertj-core:${assertj}")
+
```

Comment:
I am a bit concerned about pulling dependencies on other extensions and on test libraries into the `*-spi` directory. But maybe it's acceptable if it's only test fixture implementations... @paullatzelsperger what do you think?

Reply:
@bscholtes1A @wolf4ood I would slightly lean toward ben's position, i.e. removing the dependency onto `dataspace-authority-spi`, because we're only using a simple factory method.